llvm/lib/Target/DirectX/DXContainerGlobals.cpp

```diff
@@ -28,7 +28,7 @@ using namespace llvm::dxil;
 namespace {
 class DXContainerGlobals : public llvm::ModulePass {
 
-  GlobalVariable *getShaderFlags(Module &M);
+  GlobalVariable *getShaderFeatureInfo(Module &M);
```

Comment:
This name is confusing. Why are we calling a flag encoding "feature info"?

Reply:
So just getFeatureInfo(Module &M)?

Reply:
"feature info" doesn't mean anything to me when I read the code. The GlobalVaraible returned here, and the `uint64_t` returned in other places they aren't a "feature info". They are "feature flags".

Reply:
Changed to getFeatureFlags.